### ISSUE TITLE
doesnt work

### ISSUE THREAD

**Issue by xtendtech** [callmearta/clubhouse-desktop#73]
It doesnt send validation messages.I dont see any update in last 1 year .. Is it dead?

**Comment by Cybernetic1** [comment]
Same problem here, I don't get any verification message.

**Comment by callmearta** [comment]
Hey guys,
I'm aware of the problem and since about last year when Clubhouse changed some of it's apis, I started developing it furthermore. 
Though the code is here and you can edit it on your own to make the magic happen again but I won't be working on it anymore. There's no rational reason, no income, no credit and no help. I also saw someone selling my app on Microsoft Store and no matter how much I tried to get it removed, I got no response in return.
So feel free to edit the code and open a pull request.

**Comment by randolf** [comment]
After reading your message, I looked on the Microsoft Store and I see that the pirated app has only low ratings with a lot of complaints.  One comment calls it a scam.  It seems that Microsoft really doesn't care, which isn't surprising given their history anyway.

**Comment by callmearta** [comment]
Yes that's the reason i stopped updating the repo. Because whoever did this doesn't know how to fix problems and now that apis of clubhouse have changed, there's nothing he can do about it.
